pkg/operator/config.go

```diff
@@ -57,10 +57,11 @@ type Config struct {
 	LocalHost string
 
 	// Label and field selectors for resource watchers.
-	PromSelector            LabelSelector
-	AlertmanagerSelector    LabelSelector
-	ThanosRulerSelector     LabelSelector
-	SecretListWatchSelector FieldSelector
+	PromSelector                 LabelSelector
+	AlertmanagerSelector         LabelSelector
+	ThanosRulerSelector          LabelSelector
+	SecretListWatchSelector      FieldSelector
```

Comment:
could you rename for clarity?
```suggestion
	SecretListWatchFieldSelector      FieldSelector
```

Reply:
sure, renamed.